docs/api/browser-window.md

```diff
@@ -169,6 +169,15 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is getting resized on all platforms.
+
+### Event: 'move'
```

Comment:
I couldn't find hook yet for `linux/windows` that matches the behaviour of `os x` , but if i had to do the other way around `os x` provides only `will` and `did` notifications, have to do live dragging manually. /cc @zcbenz